antlr/src/main/antlr/FSH.g4

```diff
@@ -183,12 +183,12 @@ COMMA_DELIMITED_CODES: (CODE (WS+ STRING)? WS* COMMA WS+)+ CODE (WS+ STRING)?;
 COMMA_DELIMITED_SEQUENCES: (SEQUENCE WS* COMMA WS*)+ SEQUENCE;
 
                  // NON-WHITESPACE
-SEQUENCE:           ~[ \t\r\n\f]+;
+SEQUENCE:           ~[ \t\r\n\f\u00A0]+;
```

Comment:
Can this be simplified to:
```
SEQUENCE:           ~WS+;
```

Reply:
Since this is the second time we'd use `~WS+`, should we create another fragment for `NONWS`?

Reply:
This, we can do. In places where we need exactly `~WS`, the new `NONWS` fragment will be used. But it has to be exact: we can't use these nice fragments if we need some extra characters to be matched.